[2023-07-26T11:32:37.488+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_ingestion-pq.load_gcs_to_bq manual__2023-07-26T11:23:36.281445+00:00 [queued]>
[2023-07-26T11:32:37.524+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_ingestion-pq.load_gcs_to_bq manual__2023-07-26T11:23:36.281445+00:00 [queued]>
[2023-07-26T11:32:37.526+0000] {taskinstance.py:1308} INFO - Starting attempt 2 of 2
[2023-07-26T11:32:37.609+0000] {taskinstance.py:1327} INFO - Executing <Task(GCSToBigQueryOperator): load_gcs_to_bq> on 2023-07-26 11:23:36.281445+00:00
[2023-07-26T11:32:37.658+0000] {standard_task_runner.py:57} INFO - Started process 727 to run task
[2023-07-26T11:32:37.702+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'data_ingestion-pq', 'load_gcs_to_bq', 'manual__2023-07-26T11:23:36.281445+00:00', '--job-id', '9', '--raw', '--subdir', 'DAGS_FOLDER/data_ingest_pq.py', '--cfg-path', '/tmp/tmp7s5_02vh']
[2023-07-26T11:32:37.731+0000] {standard_task_runner.py:85} INFO - Job 9: Subtask load_gcs_to_bq
[2023-07-26T11:32:37.885+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/settings.py:195: DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
  SQL_ALCHEMY_CONN = conf.get("database", "SQL_ALCHEMY_CONN")

[2023-07-26T11:32:38.172+0000] {task_command.py:410} INFO - Running <TaskInstance: data_ingestion-pq.load_gcs_to_bq manual__2023-07-26T11:23:36.281445+00:00 [running]> on host f04213f91f62
[2023-07-26T11:32:38.870+0000] {taskinstance.py:1547} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='data_ingestion-pq' AIRFLOW_CTX_TASK_ID='load_gcs_to_bq' AIRFLOW_CTX_EXECUTION_DATE='2023-07-26T11:23:36.281445+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-07-26T11:23:36.281445+00:00'
[2023-07-26T11:32:38.881+0000] {crypto.py:83} WARNING - empty cryptography key - values will not be stored encrypted.
[2023-07-26T11:32:38.906+0000] {base.py:73} INFO - Using connection ID 'google_cloud_default' for task execution.
[2023-07-26T11:32:38.940+0000] {gcs_to_bigquery.py:373} INFO - Using existing BigQuery table for storing data...
[2023-07-26T11:32:39.273+0000] {gcs_to_bigquery.py:377} INFO - Executing: {'load': {'autodetect': False, 'createDisposition': 'CREATE_IF_NEEDED', 'destinationTable': {'projectId': 'snappy-lattice-393408', 'datasetId': 'final_project', 'tableId': 'marketing'}, 'sourceFormat': 'PARQUET', 'sourceUris': ['gs://final_project_data_lake_snappy-lattice-393408/raw/bank_marketing.csv'], 'writeDisposition': 'WRITE_TRUNCATE', 'ignoreUnknownValues': False, 'schema': {'fields': [{'name': 'client_id', 'type': 'INTEGER', 'mode': 'NULLABLE'}, {'name': 'age', 'type': 'INTEGER', 'mode': 'NULLABLE'}, {'name': 'job', 'type': 'STRING', 'mode': 'NULLABLE'}, {'name': 'marital', 'type': 'STRING', 'mode': 'NULLABLE'}, {'name': 'education', 'type': 'STRING', 'mode': 'NULLABLE'}, {'name': 'defaultloan', 'type': 'STRING', 'mode': 'NULLABLE'}, {'name': 'housingloan', 'type': 'STRING', 'mode': 'NULLABLE'}, {'name': 'loan', 'type': 'STRING', 'mode': 'NULLABLE'}, {'name': 'contact', 'type': 'STRING', 'mode': 'NULLABLE'}, {'name': 'month', 'type': 'STRING', 'mode': 'NULLABLE'}, {'name': 'day_of_week', 'type': 'STRING', 'mode': 'NULLABLE'}, {'name': 'duration', 'type': 'INTEGER', 'mode': 'NULLABLE'}, {'name': 'campaign', 'type': 'INTEGER', 'mode': 'NULLABLE'}, {'name': 'pdays', 'type': 'INTEGER', 'mode': 'NULLABLE'}, {'name': 'previous', 'type': 'INTEGER', 'mode': 'NULLABLE'}, {'name': 'poutcome', 'type': 'STRING', 'mode': 'NULLABLE'}, {'name': 'emp_var_rate', 'type': 'FLOAT', 'mode': 'NULLABLE'}, {'name': 'cons_price_idx', 'type': 'FLOAT', 'mode': 'NULLABLE'}, {'name': 'cons_conf_idx', 'type': 'FLOAT', 'mode': 'NULLABLE'}, {'name': 'euribor3m', 'type': 'FLOAT', 'mode': 'NULLABLE'}, {'name': 'nr_employed', 'type': 'FLOAT', 'mode': 'NULLABLE'}, {'name': 'subcribed', 'type': 'STRING', 'mode': 'NULLABLE'}]}}}
[2023-07-26T11:32:39.329+0000] {bigquery.py:1592} INFO - Inserting job ***_data_ingestion_pq_load_gcs_to_bq_2023_07_26T11_23_36_281445_00_00_0a121fafe6002d045bb372cdd2541ede
[2023-07-26T11:32:41.407+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/google/cloud/transfers/gcs_to_bigquery.py", line 436, in execute
    job.result(timeout=self.result_timeout, retry=self.result_retry)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/cloud/bigquery/job/base.py", line 922, in result
    return super(_AsyncJob, self).result(timeout=timeout, **kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/google/api_core/future/polling.py", line 261, in result
    raise self._exception
google.api_core.exceptions.NotFound: 404 Not found: URI gs://final_project_data_lake_snappy-lattice-393408/raw/bank_marketing.csv
[2023-07-26T11:32:41.476+0000] {taskinstance.py:1350} INFO - Marking task as FAILED. dag_id=data_ingestion-pq, task_id=load_gcs_to_bq, execution_date=20230726T112336, start_date=20230726T113237, end_date=20230726T113241
[2023-07-26T11:32:41.578+0000] {standard_task_runner.py:109} ERROR - Failed to execute job 9 for task load_gcs_to_bq (404 Not found: URI gs://final_project_data_lake_snappy-lattice-393408/raw/bank_marketing.csv; 727)
[2023-07-26T11:32:41.694+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-07-26T11:32:41.784+0000] {taskinstance.py:2653} INFO - 0 downstream tasks scheduled from follow-on schedule check
